assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -49,6 +49,9 @@ import WidgetCellWrapper from './WidgetCellWrapper';
 import useViewOnly from '../../../hooks/useViewOnly';
 import { CORE_USER } from '../../datastore/user/constants';
 import useLatestIntersection from '../../../hooks/useLatestIntersection';
+import NewBadge from '../../../components/NewBadge';
+import { WEEK_IN_SECONDS } from '../../../util';
+import { useDispatch } from '@wordpress/data';
```

Comment:
Just caught that this was used 😅 

It won't cause any errors, but we try to import from `googlesitekit-data` instead. Maybe we should add an ESLint rule about this; I know it autocompletes in VSCode a lot. I'll see if I can improve the suggestions in #8769. 🤔